packages/twenty-server/src/engine/integrations/message-queue/jobs.module.ts

```diff
@@ -65,6 +65,15 @@ import { DeleteConnectedAccountAssociatedCalendarDataJob } from 'src/modules/mes
     UserWorkspaceModule,
     WorkspaceDataSourceModule,
     RecordPositionBackfillModule,
```

Comment:
Next step should be to improve this job module, we still have to inject everything here, we could at least delegate this to sub modules